sites/public/src/forms/finder/FinderRentalCosts.tsx

```diff
@@ -11,6 +12,7 @@ const FinderRentalCosts = (props: {
   minRent: number
   maxRent: number
 }) => {
+  const dollarSign = useRouter().locale !== "ar"
```

Comment:
Not sure here, but I think we may want the $ in all languages, since that's the currency of the listings?

Reply:
@emilyjablonski So this decision came from the google sheet since the translated Arabic includes "in USD" in the copy in-place of the dollar sign. 
<img width="790" alt="image" src="https://user-images.githubusercontent.com/53269332/197621557-a14fa80e-a645-47f4-b3a0-61cb93a22205.png">

Reply:
Makes sense! Hope these aren't confusing for users, but super outside the scope of this PR.